edu.cuny.hunter.log.core/src/edu/cuny/hunter/log/core/analysis/LogInvocation.java

```diff
@@ -1,5 +1,6 @@
 package edu.cuny.hunter.log.core.analysis;
 
+import java.lang.module.ResolvedModule;
```

Comment:
This is unneeded AFAIK and is failing the build.

Reply:
Let me fix it.